modules/common/src/main/scala/notebook/front/widgets/magic/Magic.scala

```diff
@@ -27,42 +26,50 @@ trait MagicRenderPoint { me =>
 case class StringPoint(string:String, headers:Seq[String]=Seq("string value")) extends MagicRenderPoint {
   val values  = Seq(string)
 }
-case class AnyPoint(any:Any, header:Option[String]=None) extends MagicRenderPoint {
-  val headers = any match {
+
+case class AnyPoint(point:Any, header:Option[String]=None) extends MagicRenderPoint {
+
+  private val dateFormat = new SimpleDateFormat("EEE MMM dd HH:mm:ss yyyy Z")
+
+  override val headers = point match {
       case null           => Seq(header.getOrElse("Null"))
-      case v: Int         => Seq(header.getOrElse("Int"))
-      case v: Float       => Seq(header.getOrElse("Float"))
-      case v: Double      => Seq(header.getOrElse("Double"))
-      case v: Long        => Seq(header.getOrElse("Long"))
-      case v: BigDecimal  => Seq(header.getOrElse("BigDecimal"))
-      case v: String      => Seq(header.getOrElse("String"))
-      case v: Boolean     => Seq(header.getOrElse("Boolean"))
-      case v: SqlDate     => Seq(header.getOrElse("Date"))
-      case v: SqlTimestamp => Seq(header.getOrElse("Timestamp"))
-      case v: Date        => Seq(header.getOrElse("Date"))
-      case v: Geometry    => Seq(header.getOrElse("Geometry"))
-      case v: GeoJSON     => Seq(header.getOrElse("GeoJSON"))
-      case v: Any         => Reflector.toFieldNameArray(any)
+      case _: Int         => Seq(header.getOrElse("Int"))
+      case _: Float       => Seq(header.getOrElse("Float"))
+      case _: Double      => Seq(header.getOrElse("Double"))
+      case _: Long        => Seq(header.getOrElse("Long"))
+      case _: BigDecimal  => Seq(header.getOrElse("BigDecimal"))
+      case _: String      => Seq(header.getOrElse("String"))
+      case _: Boolean     => Seq(header.getOrElse("Boolean"))
+      case _: SqlDate     => Seq(header.getOrElse("Date"))
+      case _: SqlTimestamp => Seq(header.getOrElse("Timestamp"))
```

Comment:
should we use a real/full type names for `SqlDate` and `SqlTimestamp` (like for other data types)?

Reply:
Do you mean string names? I think it would make sense. I haven't touched that part. We can change it now if you want

Reply:
I mean:
```scala
case _: SqlDate     => Seq(header.getOrElse("SqlDate"))
case _: SqlTimestamp => Seq(header.getOrElse("SqlTimestamp"))
```

Reply:
Ok so I understood it correctly. I will change and push this shortly.